src/Icon/Icon.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import FontAwesome from 'react-fontawesome'
+import PatternflyIcon from './PatternflyIcon'
+
+const Icon = ({ type, ...props }) => {
+  if (type === 'fa') {
+    return <FontAwesome {...props} />
+  }
+  if (type === 'pf') {
+    return <PatternflyIcon {...props} />
+  }
```

Comment:
This is fine as-is, but to avoid the repetitive return statement you could do:

```
const IconComponent = (type === 'fa' && FontAwesome || type === 'pf && PatternflyIcon);
if (IconComponent) {
  return <IconComponent {...props} />
}
throw new Error(...)
```

Reply:
I like it. Less code and accomplishes the same for now. I'll wait until @jgiardino responds and update the PR.